new/vcloud-manager-blueprint.yaml.template

```diff
@@ -0,0 +1,19 @@
+management_network_name: some_managements_network
```

Comment:
@0lvin @ran-z don't we want to provide the same inputs file format with documentation and all?
Also, can we please rename this file to correspond with the current convention? (i.e. vcloud-manager-blueprint-inputs.yaml)